htlcswitch/switch_test.go

```diff
@@ -4290,14 +4290,15 @@ func TestSwitchDustForwarding(t *testing.T) {
 	// We'll test that once the default threshold is exceeded on the
 	// Alice -> Bob channel, either side's calls to SendHTLC will fail.
 	//
-	// Alice will send 357 HTLC's of 700sats. Bob will also send 357 HTLC's
-	// of 700sats. If either side attempts to send a dust HTLC, it will
-	// fail so amounts below 800sats will breach the dust threshold.
+	// Alice will send 354 HTLC's of 700sats. Bob will also send 354 HTLC's
+	// of 700sats.
+	numHTLCs := 354
```

Comment:
does the change in the magic number result in anything significant?

Reply:
nope just makes the test pass since the definition has been expanded